### ISSUE TITLE
PHP Parse error

### ISSUE THREAD

**Issue by devincowan** [aarenson/PHPCap#11]
PHP Parse error:  syntax error, unexpected 'class' (T_CLASS), expecting identifier (T_STRING) or variable (T_VARIABLE) or '{' or '$' in PHPCap/src/RedCapProject.php on line 2013

**Comment by mullen2** [comment]
Hi Devin,

Thanks for the feedback. I have looked into this issue, and I can't see a problem with the code.

To figure out the cause of the problem, I need more information.

What version of PHP are you using? PHPCap requires version 5.6. If you are using an earlier version, it appears you might get this error.

Jim

**Comment by devincowan** [comment]
Thanks for the Note Jim!

You are quite right, I was on a machine with an even older version of PHP when I got that error.

Many thanks for your work to maintain this repo—it’s been incredibly helpful to my research team!

Best,
Devin

On Oct 8, 2018, at 6:03 AM, Jim Mullen <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi Devin,

Thanks for the feedback. I have looked into this issue, and I can't see a problem with the code.

To figure out the cause of the problem, I need more information.

What version of PHP are you using? PHPCap requires version 5.6. If you are using an earlier version, it appears you might get this error.

Jim
